.github/workflows/test-helm-chart.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
 
       - name: Install our helm chart
         run: |
-          helm upgrade --install -f ci/values.yaml --timeout 10m --create-namespace --namespace posthog posthog ./charts/posthog --wait --wait-for-jobs
+          helm upgrade --install -f ci/values/k3s.yaml --timeout 10m --create-namespace --namespace posthog posthog ./charts/posthog --wait --wait-for-jobs
```

Comment:
Curious why the renaming to k3s?

Reply:
because the original `ci/values.yaml` was used for CI testing against a [k3s](https://k3s.io/) cluster

Reply:
Awesome, makes sense - thank you.